ipld/hamt/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+Changes to the reference FVM's HAMT implementation.
+
+## [Unreleased]
+
+## 0.8.0 [2023-08-18)
+
+Breaking Changes:
+
+- Deprecate default bitwidths in the HAMT
+  - Users must now always specify the bitwidth
+- TODO: Something about #1808?
```

Comment:
I don't wanna go diving for context since the PR description doesn't say much here, but if someone wants to add something here, please do. If not will just drop.